content/getting-started/sections/atom-basics.md

```diff
@@ -95,14 +95,23 @@ You can run the `atom` command with one or more file paths to open up those file
 
 ``` command-line
 $ atom --help
-> Atom Editor v1.8.0
+> Atom Editor v1.36.0
 
-> Usage: atom [options] [path ...]
+> Usage:
+>  atom
+>  atom [options] [path ...]
+>  atom file[:line[:column]]
 
-> One or more paths to files or folders may be specified. If there is an
-> existing Atom window that contains all of the given folders, the paths
-> will be opened in that window. Otherwise, they will be opened in a new
-> window.
+> If no arguments are given and no Atom windows are already open, restore all windows
+> from the previous editing session. Use "atom --new-window" to open a single empty
+> Atom window instead.
+
+> If no arguments are given and at least one Atom window is open, open a new, empty
+> Atom window.
+
+> One or more paths to files or folders may be specified. All paths will be opened
+> in a new Atom window. Each file may be opened at the desired line (and optionally
```

Comment:
They won't be opened in a new Atom window if the files or folders are within one of the project roots of an already open Atom window, correct?